### ISSUE TITLE
Fix Composer 2 compatibility

### ISSUE THREAD

**Issue by brandonkelly** [tomdiggle/craft-gatekeeper#12]
When installing this plugin with Composer 1, the following deprecation notice is output:

> Deprecation Notice: Class `tomdiggle\gatekeeper\assetbundles\Gatekeeper\GatekeeperAsset` located in `./vendor/tomdiggle/craft-gatekeeper/src/assetbundles/gatekeeper/GatekeeperAsset.php` does not comply with psr-4 autoloading standard\. It will not autoload anymore in Composer v2\.0\.

Composer 2 was released on October 24, and is now the default version that will be installed, so it’s critical that this gets resolved ASAP, or people will start getting `Class not found` errors.

To fix, run the following command:

```bash
> git mv src/assetbundles/gatekeeper src/assetbundles/Gatekeeper
```

Then commit your change and [tag a new release](https://craftcms.com/docs/3.x/extend/plugin-store.html#plugin-releases).

_(Note that you must use the `git mv` command, as Git tends to not notice case-sensitive file renames otherwise.)_

**Comment by tomdiggle** [comment]
This should now be fixed in version 1.2.1. Thank you.

**Comment by brandonkelly** [comment]
Thanks! I had to update the plugin in the Plugin Store manually, as it looks like we no longer have a connection to your GitHub repo. Please log into https://id.craftcms.com/ and reconnect your GitHub account to fix.
